docs/DeveloperGuide.md

```diff
@@ -608,7 +608,9 @@ NUS students
 |v1.0|careless user|edit the data easily|correct any mistakes I inputted wrongly/
```

Comment:
At section 4.1:
Would the diagram be more complete if there is a lifeline for the caller of ModTracker?
![Screenshot 2020-10-29 142654](https://user-images.githubusercontent.com/48957360/97533577-68be9580-19f3-11eb-9936-cca44c7ebffa.png)

Reply:
At section 4.5:
Should there be a ':' in front of the class name?

![Screenshot 2020-10-29 144201](https://user-images.githubusercontent.com/48957360/97534665-59d8e280-19f5-11eb-9f1a-8aba45628ee0.png)

Reply:
At section 4.7:
Should :TaskList return control to :Parser instead of User since it was called by :Parser? Also I thought the Parser is called by :Ui instead of directly by User?
![Screenshot 2020-10-29 144902](https://user-images.githubusercontent.com/48957360/97535285-61e55200-19f6-11eb-85f5-b1c1c95c9bfe.png)